src/pyramid/security.py

```diff
@@ -114,7 +92,7 @@ def principals_allowed_by_permission(context, permission):
     in effect, return a sequence of :term:`principal` ids that possess
     the permission in the ``context``.  If no authorization policy is
     in effect, this will return a sequence with the single value
-    :mod:`pyramid.security.Everyone` (the special principal
+    :mod:`pyramid.authorization.Everyone` (the special principal
```

Comment:
Should the `principals_allowed_by_permission` stay in `pyramid.security`?

Reply:
no it's fine as-is. The function is deprecated and there is not a way to make it work without changes going forward as it currently depends on an authorization policy. I can't put a version in pyramid.authorization that actually works going forward.